### PR TITLE
Update partitioned-table.md

### LINKED ISSUE #10817: INTERVAL partitioning does not support multi-column RANGE COLUMNS partitioning
## Error Report

Please answer the following questions before submitting your issue. Thanks!

1. What is the URL/path of the document related to this issue?
https://docs.pingcap.com/tidb/dev/partitioned-table#interval-partitioning-details-and-limitations

2. How would you like to improve it?
Add that only a single partitioning column is supported for RANGE COLUMNS partitioning

### PR DESCRIPTION
**Pull request by hey-hoho**
### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs) that's required for repo owners to accept my contribution.

### What is changed, added or deleted? (Required)
fix #10817 

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions](https://github.com/pingcap/docs/blob/master/CONTRIBUTING.md#guideline-for-choosing-the-affected-versions).

- [x] master (the latest development version)
- [x] v7.6 (TiDB 7.6 versions)
- [x] v7.5 (TiDB 7.5 versions)
- [x] v7.4 (TiDB 7.4 versions)
- [x] v7.3 (TiDB 7.3 versions)
- [x] v7.1 (TiDB 7.1 versions)
- [x] v6.5 (TiDB 6.5 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: partitioned-table.md [hunk 1]
```diff
@@ -295,7 +295,7 @@ ALTER TABLE table_name LAST PARTITION LESS THAN (<expression>)
 - The INTERVAL partitioning feature only involves the `CREATE/ALTER TABLE` syntax. There is no change in metadata, so tables created or altered with the new syntax are still MySQL-compatible.
 - There is no change in the output format of `SHOW CREATE TABLE` to keep MySQL compatibility.
 - The new `ALTER` syntax applies to existing tables conforming to INTERVAL. You do not need to create these tables with the `INTERVAL` syntax.
-- For `RANGE COLUMNS`, only integer, date, and datetime column types are supported.
+- For `RANGE COLUMNS`, only one column that is integer, date, and datetime types is supported as partitioning key.
```
Comment (dveeden) [inline partitioned-table.md:298]:
```suggestion
- For `RANGE COLUMNS`, only one column that is an integer, date, and datetime types is supported as partitioning key.
```

File: partitioned-table.md [hunk 2]
```diff
@@ -295,7 +295,7 @@ ALTER TABLE table_name LAST PARTITION LESS THAN (<expression>)
 - The INTERVAL partitioning feature only involves the `CREATE/ALTER TABLE` syntax. There is no change in metadata, so tables created or altered with the new syntax are still MySQL-compatible.
 - There is no change in the output format of `SHOW CREATE TABLE` to keep MySQL compatibility.
 - The new `ALTER` syntax applies to existing tables conforming to INTERVAL. You do not need to create these tables with the `INTERVAL` syntax.
-- For `RANGE COLUMNS`, only integer, date, and datetime column types are supported.
+- For `RANGE COLUMNS`, only one column that is an integer, date, or datetime types is supported as partitioning key.
```
Comment (qiancai) [inline partitioned-table.md:298]:
```suggestion
- For `RANGE COLUMNS` partitioning, only one column that is in the integer, date, or datetime type can be used as the partitioning key.
```

File: partitioned-table.md
Comment (qiancai) [inline partitioned-table.md:1]: Besides L298, please also update the L172 to fix the issue:

image

Comment (hey-hoho) [reply partitioned-table.md:1]: range column partitioning can use mutilple columns, but it cannot be used together with range interval partitioning
.

Comment (qiancai) [reply partitioned-table.md:1]: yes, you're right.👍 Thanks for the explaination.

File: partitioned-table.md [hunk 3]
```diff
@@ -295,7 +295,7 @@ ALTER TABLE table_name LAST PARTITION LESS THAN (<expression>)
 - The INTERVAL partitioning feature only involves the `CREATE/ALTER TABLE` syntax. There is no change in metadata, so tables created or altered with the new syntax are still MySQL-compatible.
 - There is no change in the output format of `SHOW CREATE TABLE` to keep MySQL compatibility.
 - The new `ALTER` syntax applies to existing tables conforming to INTERVAL. You do not need to create these tables with the `INTERVAL` syntax.
-- For `RANGE COLUMNS`, only integer, date, and datetime column types are supported.
+- For `RANGE COLUMNS` partitioning, only one column that is in the integer, date, or datetime type can be used as the partitioning key.
```
Comment (qiancai) [inline partitioned-table.md:298]:
```suggestion
- If you want to use the `INTERVAL` syntax for `RANGE COLUMNS` partitioning, only one column that is in the integer, date, or datetime type can be specified as the partitioning key.
```

**Comment by mjonss** [comment]
This should not be added to v6.5 or later, since it was added in 6.5. v6.1 does support single column RANGE COLUMNS partitioning, but it does not seem to be documented (also not sure if it was considered GA or not in v6.1). Looking at the first PR, https://github.com/pingcap/docs/pull/10819, it was proposed only for v6.3/v6.4, so I assume RANGE COLUMNS was not considered GA in v6.1.

So unless we update v6.1, I think we can close this pr and the associated bug instead.

**Review by mjonss — CHANGES_REQUESTED** [review]
I made a mistake, not taking the INTERVAL part into account, only the RANGE COLUMNS multi-columns one.
~~Multi-column RANGE COLUMN is supported since v6.5, so please consider [this comment](https://github.com/pingcap/docs/pull/16123#issuecomment-1900075811).~~

**Comment by mjonss** [comment]
Multi-column RANGE COLUMNs was added in v6.3, so the single column RANGE COLUMNs docs update would only apply to v6.1, which don't even mention RANGE COLUMN, so I think it is safe to close this PR and [issue](https://github.com/pingcap/docs/issues/10817).

**Comment by qiancai** [comment]
It seems that [INTERVAL partitioning does not support multi-column RANGE COLUMNS partitioning · Issue #10817 · pingcap/docs](https://github.com/pingcap/docs/issues/10817) was reported after the following two v6.3 PRs are merged:

- [table partition: add INTERVAL partitioning by mjonss · Pull Request #9751 · pingcap/docs](https://github.com/pingcap/docs/pull/9751)
- [table partition: add RANGE COLUMNS partitioning by mjonss · Pull Request #10385 · pingcap/docs](https://github.com/pingcap/docs/pull/10385)

**Review by mjonss — APPROVED** [review]
LGTM, with the suggestion from @qiancai.
Sorry I previously missed the INTERVAL part of the issue.

File: partitioned-table.md [hunk 4]
```diff
@@ -295,7 +295,7 @@ ALTER TABLE table_name LAST PARTITION LESS THAN (<expression>)
 - The INTERVAL partitioning feature only involves the `CREATE/ALTER TABLE` syntax. There is no change in metadata, so tables created or altered with the new syntax are still MySQL-compatible.
 - There is no change in the output format of `SHOW CREATE TABLE` to keep MySQL compatibility.
 - The new `ALTER` syntax applies to existing tables conforming to INTERVAL. You do not need to create these tables with the `INTERVAL` syntax.
-- For `RANGE COLUMNS`, only integer, date, and datetime column types are supported.
+- If you want to use the `INTERVAL` syntax for `RANGE COLUMNS` partitioning, only one column that is in the integer, date, or datetime type can be specified as the partitioning key.
```
Comment (qiancai) [inline partitioned-table.md]:
```suggestion
- If you want to use the `INTERVAL` syntax for `RANGE COLUMNS` partitioning, only a single column that is in the integer, date, or datetime type can be specified as the partitioning key.
```

File: partitioned-table.md [hunk 5]
```diff
@@ -295,7 +295,7 @@ ALTER TABLE table_name LAST PARTITION LESS THAN (<expression>)
 - The INTERVAL partitioning feature only involves the `CREATE/ALTER TABLE` syntax. There is no change in metadata, so tables created or altered with the new syntax are still MySQL-compatible.
 - There is no change in the output format of `SHOW CREATE TABLE` to keep MySQL compatibility.
 - The new `ALTER` syntax applies to existing tables conforming to INTERVAL. You do not need to create these tables with the `INTERVAL` syntax.
-- For `RANGE COLUMNS`, only integer, date, and datetime column types are supported.
+- If you want to use the `INTERVAL` syntax for `RANGE COLUMNS` partitioning, only a single column that is in the integer, date, or datetime type can be specified as the partitioning key.
```
Comment (hfxsd) [inline partitioned-table.md:298]:
```suggestion
- If you want to use the `INTERVAL` syntax for `RANGE COLUMNS` partitioning, you can only specify a single column that is in the `INTEGER`, `DATE`, or `DATETIME` type as the partitioning key.
```

**Pull request merged**